packages/core/src/components/tag/tag.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, Host, Prop, h } from '@stencil/core'
+import { IconProps } from '../../icons'
+
+@Component({
+  tag: 'atom-tag',
+  styleUrl: 'tag.scss',
+  shadow: true,
+})
+export class AtomTag {
+  @Prop() type: 'success' | 'danger' | 'warning' | 'dark' | 'light' | 'neutral'
```

Comment:
(block) shouldn't this be `color`? `type` has a different objective, it conveys the idea of changing behavior

Reply:
@felipefialho change here https://github.com/juntossomosmais/atomium/pull/261/commits/4a1774b59dea82172be44998e43834acc5f3d06b